etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiFhirResource.java

```diff
@@ -16,4 +16,9 @@ public HapiFhirResource(IBaseResource innerResource) {
     public IBaseResource getUnderlyingResource() {
         return innerResource;
     }
+
+    @Override
+    public String getFhirResourceId() {
+        return innerResource.getIdElement().getIdPart();
```

Comment:
Any idea why it shows that this method is not covered by the tests in HapiOrderTest and HapiResultTest?

Reply:
I merged this before seeing this was commented. Let's keep track of test coverage for this and make another PR to fix it if necessary.